.github/workflows/build.yml

```
@@ -0,0 +1,252 @@
+    needs: [build, check-version-and-tag]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+
+      - name: Fetch All Tags
+        run: git fetch --tags
+
+      - name: Verify Created Tag
+        id: verified_version
+        run: |
```

actionlint (1.7.7)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)